pkg/rancher-desktop/backend/lima.ts

```diff
@@ -423,16 +425,37 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
 
     const images = currentConfig.images.map(i => path.basename(i.location));
     // We had a typo in the name of the image; it was "alpline" instead of "alpine".
-    const versionMatch = images.map(i => /^alpl?ine-lima-v([0-9.]+)-/.exec(i)).find(defined);
+    // Image version may have a '+rd1' (or '.rd1') suffix after the upstream semver version.
+    const versionMatch = images.map(i => /^alpl?ine-lima-v([0-9.]+)(?:[+.]rd(\d+))?-/.exec(i)).find(defined);
     const existingVersion = semver.coerce(versionMatch ? versionMatch[1] : null);
+    const existingRDVersion = versionMatch ? versionMatch[2] : undefined;
```

Comment:
```suggestion
    const existingRDVersion = versionMatch?.[2];
```
(Similarly for `existingVersion`)

Reply:
Done